api/swagger.yaml

```diff
@@ -151,9 +146,8 @@ definitions:
   AppInit:
     type: object
     properties:
-      id:
-        type: integer
-        format: int64
+      appId:
+        type: string
```

Comment:
Can I ask why the switch from integer to string & how this is being generated?

Reply:
> @davidffrench yea seen that but its a pet hate of mine with api docs that you never get the full route in the same place

@davidffrench @austincunningham I don't think `/api` is the base URL - it is the base path. Something like `http://my-server-addr.com` would be the base URL.

Reply:
OF course, the appId will be in the format `A!B2C3D4E5.com.domainname.applicationname`